tests/integration/base_block_test.go

```diff
@@ -231,5 +232,58 @@ spec:
       - name: block-persistent-storage
         persistentVolumeClaim:
           claimName: ` + blockName + `
+          readOnly: ` + strconv.FormatBool(readOnly) + `
       restartPolicy: Never`
 }
+
+func getBlockStatefulSetDefintion(statefulsetName, podname, StorageClassName string) string {
```

Comment:
i would like us to get in the habit of using the k8s api instead of these yamls everywhere, if we could at least start using it for new code.

Reply:
using  k8s API now.